### DISCUSSION TITLE
Icon request: Fox

### DISCUSSION THREAD

**Discussion by veringm** [FortAwesome/Font-Awesome#21179] [Icon Requests]
### What can it be used for?
To be displayed in an mobile-app for reporting various animal-crew sizes. Already using the existing fontawesome pro animal icons.

The user can easily find the right animal-type to update (icon follow by helper text).

### Example image (optional)
https://www.google.com/search?q=fox&safe=active&rlz=1C1GCEU_daDK819DK820&source=lnms&tbm=isch&sa=X&ved=0ahUKEwjBuc7e4bviAhVawsQBHQyyDAUQ_AUIDigB&biw=1536&bih=859

### Icon request checklist

- [X] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [X] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [X] I have included at least one example image if it could be helpful (optional)
- [X] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺)

**Comment by hexandcube** [comment]
That would be great

**Comment by shyredfox** [comment]
I would also love to see a fox icon (that isn't the firefox one).

**Comment by ghost** [comment]
Why is this not a thing yet? You even have a crow icon, so why not a fox?

**Comment by MarksT22** [comment]
Related Unicode emoji: 🦊 U+1F98A

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by hexandcube** [comment]
Awesome! Just hope that the basic variants will be available in the free kit

**Comment by shyredfox** [comment]
yay! Looking forward to the fox icon! <3
